apps/client/src/shared/components/insurance-title/insurance-title.tsx

```diff
@@ -0,0 +1,17 @@
+import { titleVariants } from './insurance-title.css';
+
+interface InsuranceTitleProps {
+  company: string; //api 연동
+  name: string; //api 연동
```

Comment:
```suggestion
  company: string; // TODO 명세 필드명 반영
  name: string;  // TODO 명세 필드명 반영 
```

이 더 좋은 주석인 것 같아요 제가 앞에 // TODO 로 형식을 통일하자는 이유는 추후에 // TODO 로 검색을 해서 빠트린 내용을 찾기 위해서 입니다